docs/usage/modules/manager/index.md

```diff
@@ -18,38 +18,38 @@ The goal of Renovate is to detect and maintain all third-party dependencies in y
 
 ### File Matching
 
-Most managers have a default `fileMatch` array.
-The `fileMatch` array has regular expression strings that match against the repository file list.
+Most managers have a default `filePatterns` array.
+The `filePatterns` array has regular expression or glob pattern that matches against the repository file list.
 
-#### Managers with no default fileMatch
+#### Managers with no default filePatterns
 
-Some managers have no default `fileMatch` regular expression, because they have no filename convention that would let Renovate intelligently filter them.
-In such a case, the manager will be disabled until you create a `fileMatch` regular expression, e.g. like the following:
+Some managers have no default `filePatterns`, because they have no filename convention that would let Renovate intelligently filter them.
+In such a case, the manager will be disabled until you create a `filePatterns` regular expression, e.g. like the following:
 
 ```json
 {
   "kubernetes": {
-    "fileMatch": ["^config/.*\\.yaml$"]
+    "filePatterns": ["/^config/.*\\.yaml$/"]
   }
 }
 ```
 
-#### Extending a manager's default fileMatch
+#### Extending a manager's default filePatterns
 
-If the default `fileMatch` regular expression for a manager does not match against one of your relevant files, you can _extend_ the existing regular expression(s) by configuring a manager's `fileMatch` like in this example:
+If the default `filePatterns` for a manager does not match against one of your relevant files, you can _extend_ the existing pattern(s) by configuring a manager's `filePatterns` like in this example:
 
 ```json
 {
   "dockerfile": {
-    "fileMatch": ["does-not-look-like-a-docker-file"]
+    "filePatterns": ["does-not-look-like-a-docker-file"]
```

Comment:
```suggestion
    "filePatterns": ["a-weird-name-for-a-dockerfile"]
```

How about something like this? The example is about a user setting their own weird Dockerfile name.

Reply:
Both seems fine